AWS/legos/aws_list_expiring_access_keys/aws_list_expiring_access_keys.py

```diff
@@ -17,9 +19,14 @@ class InputSchema(BaseModel):
 def aws_list_expiring_access_keys_printer(output):
     if output is None:
         return
-    pprint.pprint(output)
+    if isinstance(output, CheckOutput):
```

Comment:
and why is this isinstance check, a check should always return CheckOutput object

Reply:
Also as we discussed this with Abhishek, printing of the check lego should be inside the infra.

Reply:
This will be discussed by @amit-chandak-unskript